apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html

```diff
@@ -172,26 +186,28 @@ <h3 class="modal-title pull-left">{{ localeData?.rejection_reason }}</h3>
             </div>
         </div>
         <div class="form-group">
-            <label for="rejectReason">{{ localeData?.reason_label }}</label> <sup>*</sup>
-            <textarea
-                name="rejectReason"
-                required
-                id="rejectReason"
-                [formControl]="rejectReason"
-                class="textarea form-control"
-                [placeholder]="localeData?.reason_placeholder"
-            ></textarea>
+            <mat-form-field appearance="fill" class="w-100">
+                <label for="rejectReason">{{ localeData?.reason_label }}</label> <sup>*</sup>
+                <textarea
+                    matInput
+                    name="rejectReason"
+                    id="rejectReason"
+                    [formControl]="rejectReason"
+                    class="textarea form-control mat-field-border mt-2"
+                    [placeholder]="localeData?.reason_placeholder"
+                ></textarea>
+            </mat-form-field>
         </div>
 
-        <button type="submit" class="btn btn-success" [disabled]="!rejectReason.value" (click)="submitReject()">
+        <button mat-raised-button color="primary" [disabled]="!rejectReason.value" (click)="submitReject()">
```

Comment:
remove the color and use class name properly. @ravinderwalkover

Reply:
Done